docpages/advanced_reference/separate-events.md

```diff
@@ -51,26 +51,25 @@ Now, you'll have a nice area where you can easily see the code, without scrollin
 
 However, we've not finished yet! If you thought "How does the `main.cpp` file actually know to call this?" then, 10 points to you! It doesn't know! We need to go do that now. So, let's do exactly that.
 
-~~~~~~~~~~{.cpp}
+~~~~~~~~~~cpp
 #include <dpp/dpp.h>
-#include "listeners/message_listener.h" 
+#include "listeners/message_listener.h"
 
-int main()
-{
+int main() {
     /* Create the bot, but with our intents so we can use messages. */
-	dpp::cluster bot("token", dpp::i_default_intents | dpp::i_message_content);
+    dpp::cluster bot("token", dpp::i_default_intents | dpp::i_message_content);
 
```

Comment:
these have all been changed to space indentation against the coding style guide

Reply:
Ahhhhhhh why vscode 😭